### PR TITLE
Fix #2305 - IGV zoom length contig

### LINKED ISSUE #2305: Structural variants alignment in IGV shows entire genome
Hi,
When looking at structural variants, I can view the individual breakpoints in IGV but when I click the IGV button to view the entire SV, I get the whole genome (it used to be an overview of the region with the SV including both breakpoints). This is not case-specific and it's been like this for a couple of weeks.
It's obviously not a big issue, but I hope it can be reverted to the previous settings. Thank you!

### PR DESCRIPTION
**Pull request by dnil**
This PR fixes a bug.

**How to test**:
1. on a case with cram files available, verify that opening IGV view on the whole length of a SV variant (del, dup, etc) opens IGV on all configs
Screenshot 2021-01-26 at 15 09 24

1. after applying patch, note that IGV instead opens on the correct chromosome and region
 
Screenshot 2021-01-26 at 15 09 03

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
Ready for a quick review, me hopes! 😅

File: scout/server/blueprints/variant/templates/variant/utils.html [hunk 1]
```diff
@@ -271,7 +271,7 @@
           <input type="hidden" name="sample" value="{{case.sample_names|join(',')}}">
           <input type="hidden" name="build" value="{{case.genome_build}}">
 
-          <input type="hidden" name="contig" value="{% if zoom == "start" %}{{ variant.chromosome }}{% elif zoom == "end" %}{{ variant.end_chrom }}{% endif %}">
+          <input type="hidden" name="contig" value="{% if zoom == "start" or zoom == "length" %}{{ variant.chromosome }}{% elif zoom == "end" %}{{ variant.end_chrom }}{% endif %}">
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/utils.html:274]: you could also have if zoom in ["start", "length"], but it works!

**Review by northwestwitch — APPROVED** [review]
👍

File: scout/server/blueprints/variant/templates/variant/utils.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/utils.html]: That is cleaner - I will change!

**Pull request merged**